poor/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+file(GLOB_RECURSE POOR *.py)
+install(FILES ${POOR} DESTINATION ${DATADIR}/poor)
+# The following _should_ work and would be better than the above GLOB_RECURSE, but for some reason it installs broken symlinks as well instead of just Python files
+#install(DIRECTORY poor
+#	DESTINATION ${DATADIR}
+#	FILES_MATCHING
+#	PATTERN "*.py")
+
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/config.py"
+	COMMAND sed -e \'s|pure-maps|${APP_NAME}|g\' ${CMAKE_CURRENT_SOURCE_DIR}/paths.py > ${CMAKE_CURRENT_BINARY_DIR}/paths.py
+	COMMAND sed -e \'s|_default_basemap *=.*|_default_basemap = \"${DEFAULT_BASEMAP}\"|g\' ${CMAKE_CURRENT_SOURCE_DIR}/config.py > ${CMAKE_CURRENT_BINARY_DIR}/config.py
+	COMMAND sed -i -e \'s|_default_geocoder *=.*|_default_geocoder = \"${DEFAULT_GEOCODER}\"|g\' ${CMAKE_CURRENT_BINARY_DIR}/config.py
+	COMMAND sed -i -e \'s|_default_guide *=.*|_default_guide = \"${DEFAULT_GUIDE}\"|g\' ${CMAKE_CURRENT_BINARY_DIR}/config.py
+	COMMAND sed -i -e \'s|_default_router *=.*|_default_router = \"${DEFAULT_ROUTER}\"|g\' ${CMAKE_CURRENT_BINARY_DIR}/config.py
+	COMMAND "${PYTHON_EXECUTABLE}" ${CMAKE_CURRENT_SOURCE_DIR}/../tools/manage-keys inject ${DATADIR})
```

Comment:
Looks like key injection is not working. Replacing with `${CMAKE_CURRENT_SOURCE_DIR}/tools/manage-keys` didn't help either. From the logs, I don't see where it is run, unfortunately.

Reply:
Seems the entire script didn't run because it was treated as a string. Fixing that, I noticed a different issue where the script didn't have permission to do anything with the files in the directory passed to it.

Looking through the files it tried changing, it seems it only really needed to change values in `poor/keystore.py`. So, I've modified the script to just directly check the directory it was passed, and made sure that directory contained `keystore.py`, after which the modified file gets installed properly.

It's however worth considering to change this method entirely at some point, and use CMake's `configure_file()` instead.

Reply:
maybe we should reconsider the method, indeed. from man it looks like it will require adding API keys during compilation as options to `cmake`. Or is there a better way to do it, like from some cmake config file that is read during a build?

note that I have to inject and remove keys during testing, as a part of releasing routine (https://github.com/rinigus/pure-maps/blob/master/RELEASING.md). So, we may want to keep the both options available. 

If it is possible to use cmake with some extra configs specified in a config file (in addition to options given in cmd line), I can also adjust inject script to generate that config file and then use it for builds.